### PR TITLE
fix: Updating styles on screen reader heading changes.

### LINKED ISSUE #11869: Fix styling after Headings update for Screen Reader support
Heading tags were changed in several locations for better representation and hierarchy for the screen reader rotor feature. These changes resulted in minor styling issues that need to be corrected. The goal is to restore the original look and feel before the change. 

A list detailed of changes can be found [here](https://cxwiki.sap.com/display/spar/Headings+and+Labels) 
The [demo site](https://spartacus-demo.eastus.cloudapp.azure.com/electronics-spa/en/USD/) can be used as a reference to see what the styling and UI looked like before the changes 

### Todos:
- [ ] Review the detailed list of changes for headings 
- [ ] Restore original styling by comparing with demo site and update SCSS files as necessary 

### Additional info 
Epic Branch: `epic/a11y-sr/headings-and-labels`
Depends on #11867 to be merged in the epic branch first

### PR DESCRIPTION
**Pull request by morganm58**
Closes GH-11869

### CODE REVIEW

**Review by saad-mo — CHANGES_REQUESTED** [review]
I still see the following issues:

- Homepage: Product Titles carousel (Our best selling products & What's New) have lost their original styling. The font is much bigger. 
Screen Shot 2021-04-12 at 10 29 55 AM

- Checkout Pages: "Shipping Address", "Shipping Method", "Payment" font size is bigger 
Screen Shot 2021-04-12 at 10 27 57 AM

- Wish List Page: The product title font is too big
Screen Shot 2021-04-12 at 10 23 43 AM

File: projects/storefrontstyles/scss/components/misc/card/_card.scss [hunk 1]
```diff
@@ -11,6 +11,10 @@
     flex-grow: 2;
   }
 
+  .cx-card-title {
+    font-size: var(--cx-font-size, 1.125rem);
+  }
```
Comment (developpeurweb) [inline projects/storefrontstyles/scss/components/misc/card/_card.scss:17]: Let's avoid using CSS vars for font size and use `@include type ()` instead.

File: projects/storefrontstyles/scss/cxbase/mixins/_checkout-media-style.scss [hunk 2]
```diff
@@ -45,6 +45,7 @@
     margin-bottom: 0;
     margin-inline-start: auto;
     padding: 2.375rem 0 1.75rem 0;
+    font-size: var(--cx-font-size, 1.375rem);
   }
```
Comment (developpeurweb) [inline projects/storefrontstyles/scss/cxbase/mixins/_checkout-media-style.scss:50]: Let's avoid using CSS vars for font size and use `@include type ()` instead.

**Review by saad-mo — APPROVED** [review]
Looks good to marge (pending Miguel's comments)

**Review by developpeurweb — CHANGES_REQUESTED** [review]
Let's avoid using CSS vars for font size and use `@include type ()` instead. Other than that it looks fine. Pls notice you have a conflict that might prevent you from merging.

**Review by developpeurweb — APPROVED** [review]
LGTM

**Pull request merged**